packages/backend-app-api/src/wiring/BackendInitializer.ts

```diff
@@ -228,6 +233,16 @@ export class BackendInitializer {
       }
     }
 
+    const rootFeatureRegistry = await this.#serviceRegistry.get(
+      coreServices.rootFeatureRegistry,
+      'root',
+    );
+    if (
+      rootFeatureRegistry instanceof RootFeatureRegistryServiceImplementation
+    ) {
+      rootFeatureRegistry.setFeatures(registrations);
+    }
```

Comment:
We avoid this if possible. I'm thinking we'd do something similar to the plugin matadata service instead? Could even be that we extend that service for the feature introspection.

Reply:
Added a `InstanceMetadataService` to track this information, I don't think it quite fits in the plugin metadata service.